source/d_geometryblocks.rst

```diff
@@ -0,0 +1,190 @@
+                        "build_year": "building_build_year"
+                    },
+                    "geometry",
+                    "start",
+                    "end"
+                ],
+                "buildyear": [
+                    "geoblocks.geometry.base.GetSeriesBlock",
+                    "buildings",
+                    "building_build_year"
+                ],
+                "label": [
+                    "geoblocks.geometry.field_operations.Classify",
+                    "buildyear",
+                    [
+                        1900,
+                        1940,
+                        1970,
+                        1990
+                    ],
+                    [
+                        "A",
+                        "B",
+                        "C",
+                        "D",
+                        "E"
+                    ]
+                ],
+                "result": [
+                    "geoblocks.geometry.base.SetSeriesBlock",
+                    "buildings",
+                    "label_value",
+                    "label"
+                ]
+            },
+            "name": "result"
+        }
+    }
+
+Geometry output
+---------------
+
+Geometry outputs are stored in :doc:`labels<c_labels>`.
+Labels are always linked to your Vectors stored in the Vector Server, for instance flood risk for parcels or buildings.
+Labels are grouped in Labeltypes. The graph can be found in the source attribute of the labeltype.
+
+.. image:: /images/d_geoblocks_02.png 
+
+Individual labels (e.g. label linked to one building or parcel) can be found on the `labels endpoint <demo.lizard.net/api/v3/labels>`_.  
+Labels can be computed on the fly using the compute endpoint or a-sync using the Lizard Task Server. 
+
+Operations
+----------
+
+.. automodule:: dask_geomodeling.geometry.base
+   :members: GeometryBlock, SeriesBlock, GetSeriesBlock, SetSeriesBlock
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.aggregate`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.aggregate
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.constructive`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.constructive
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.field_operations`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.field_operations
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.geom_operations`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.geom_operations
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.parallelize`
```

Comment:
This is not allowed through the API, should be omitted

Reply:
You mean the 'parralelize'?

Reply:
Yes, parallelize, sources and sinks are internal and should not be configured through the API

Reply:
Ok, removed!